utils/types/reporting_types.go

```diff
@@ -21,7 +21,9 @@ const (
 )
 
 var (
-	DiffStatus = "diff"
+	DiffStatus     = "diff"
+	FilteredStatus = "filtered"
```

Comment:
@chandumlg is introducing a filtered job state in jobsdb, we could use the jobsdb literal instead here to avoid proliferation of equal constants